### PR TITLE
update/checkin-tests case and fixes from Jessi's test

### LINKED ISSUE #117: Logout user on 401 to refresh URL.
This is when fetching the refresh URL doesn't work

### LINKED ISSUE #130: Checkin Tests
- Test Checking in a guest

### PR DESCRIPTION
**Pull request by mhlangagc**
### Closes Issues:
- closes #130 
- closes #117 

### Pods Installed
 * No change

### CODE REVIEW

File: BigNeonStudioTests/BigNeonStudioTests.swift [hunk 1]
```diff
@@ -41,8 +41,85 @@ class BigNeonStudioTests: XCTestCase {
     
     func testTicketCheckin() {
         
+        let eventID = "39fb32a6-82f5-4d59-a901-0c8ac09734ad"
```
[the remark this answers is not in the document]
Comment (mhlangagc) [reply BigNeonStudioTests/BigNeonStudioTests.swift]: Done 👍

File: BigNeonStudioTests/BigNeonStudioTests.swift [hunk 2]
```diff
@@ -7,10 +7,9 @@ class BigNeonStudioTests: XCTestCase {
     
     private let repository = EventsApiRepository.shared
 
-    //  Environment().configuration(PlistKey.testAuthEmail)
     func testAuthentication() {
-        let email = "superuser@test.com"
-        let password = "password"
+        let email = "superuser@test.com" // Environment().configuration(PlistKey.testAuthEmail)
```
Comment (Krakaw) [inline BigNeonStudioTests/BigNeonStudioTests.swift]: Why not use the env?

File: Big Neon Core/Utilities/DateConfig.swift [hunk 3]
```diff
@@ -19,6 +19,10 @@ final public class DateConfig {
         }
         return  dateFormatter.date(from: dateFormatter.string(from: convertedDate))
     }
+    
+    public class func eventDateIsToday(eventStartDate: String) -> Bool {
```
Comment (Krakaw) [inline Big Neon Core/Utilities/DateConfig.swift]: What happens if the doors open at 11pm but the event starts at midnight? The days would be different.
Let's change the comparison to within 24 hours of the door time.

Comment (mhlangagc) [reply Big Neon Core/Utilities/DateConfig.swift]: Ok Cool. Will change it to that.

**Comment by mhlangagc** [comment]
Please check now. The fixes requested have been added in.

**Pull request merged**